agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/sampling/SamplingPercentage.java

```diff
@@ -22,4 +24,8 @@ static SamplingPercentage fixed(double percentage) {
   static SamplingPercentage rateLimited(double targetPerSecondLimit) {
     return new RateLimitedSamplingPercentage(targetPerSecondLimit, 0.1);
   }
+
+  static SamplingPercentage ingestion() {
+    return () -> USE_INGESTION_SAMPLING;
```

Comment:
```suggestion
    return () -> INGESTION_SAMPLING_PERCENTAGE;
```

Reply:
it's not boolean return type.

Reply:
not following?

Reply:
it's ok since you rename the method name to match with the constant.